base_frame: ump
map_frame: map
rate: 5.0
coords: [0, 0, 0]